SDL/Triplet.h

```diff
@@ -205,9 +195,9 @@ namespace SDL {
     tripletsInGPU.lowerModuleIndices[tripletIndex * 3 + 2] = outerOuterLowerModuleIndex;
 
     tripletsInGPU.betaIn[tripletIndex] = __F2H(betaIn);
-    tripletsInGPU.betaOut[tripletIndex] = __F2H(betaOut);
-    tripletsInGPU.pt_beta[tripletIndex] = __F2H(pt_beta);
-
+    tripletsInGPU.circleRadius[tripletIndex] = __F2H(circleRadius);
+    tripletsInGPU.circleCenterX[tripletIndex] = __F2H(circleCenterX);
+    tripletsInGPU.circleCenterY[tripletIndex] = __F2H(circleCenterY);
```

Comment:
these parts (and reverse, calls to `H2F`) should be updated to keep using floats as is

Reply:
addressed, thanks!